docs/dg/dev/third-party-module-development/develop.md

```diff
@@ -0,0 +1,159 @@
+-  Use the samples that marked as *Module Development*.
+
+
+### How you can extend Spryker functionality from your module:
+**Option 1.** Introduce an extension point in existing Spryker module, use it. (See complete example here https://docs.spryker.com/docs/dg/dev/backend-development/plugins/plugins.html#how-to-use-a-plugin-from-another-module)
+
+**Option 2.** Adding a controller, example for project-level implementation can be found here https://docs.spryker.com/docs/dg/dev/backend-development/extend-spryker/create-modules.html#display-a-random-salutation-message, core implementation will be the same but with different namespaces.
+
+**Option 3.** Use the existing Spryker module extension point.
+
+**a.** Example of existing extension point usage.
+Were we'll extend product table with an additional column **Categories** where will show the list of categories where the product is present.
+For this we'll use two extension points in the existing Spryker modules 
+- One to extend table headers with **Categories** column [ProductTableConfigurationExpanderPluginInterface](https://github.com/spryker/product-management-extension/blob/master/src/Spryker/Zed/ProductManagementExtension/Dependency/Plugin/ProductTableConfigurationExpanderPluginInterface.php)
+- And another to extend table content with the corresponding column [ProductTableDataBulkExpanderPluginInterface](https://github.com/spryker/product-management-extension/blob/master/src/Spryker/Zed/ProductManagementExtension/Dependency/Plugin/ProductTableDataBulkExpanderPluginInterface.php)
+
+a. Create a module **console dev:module:create your-company-name.product-category**. 
+
+{% info_block infoBox %}
+   Make sure that `vendor/your-company-name/product-category` folder with module data is created.
+{% endinfo_block %}
+
+**b.** Create a plugin that will extend product table headers with **Categories** column
+
+File Path: vendor/your-company-name/product-category/src/YourCompanyName/Zed/ProductCategory/Communication/Plugin/ProductManagement/ProductCategoryProductTableConfigurationExpanderPlugin.php
+
+```php 
+
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace YourCompanyName\Zed\ProductCategory\Communication\Plugin\ProductManagement;
+
+use Spryker\Zed\Gui\Communication\Table\TableConfiguration;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductTableConfigurationExpanderPluginInterface;
+
+class ProductCategoryProductTableConfigurationExpanderPlugin extends AbstractPlugin implements ProductTableConfigurationExpanderPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Expands `ProductTable` configuration with categories column.
+     *
+     * @api
```

Comment:
I gurss api tags are not relevant for 3rd party modules

Reply:
Since they will use sever the same as we do, shouldn't we keep it? (Maybe add some details into the doc about where those tags are needed)